src/assets/scss/introjs.scss

```diff
@@ -0,0 +1,103 @@
+.introjs-tooltip {
+    background-color: white;
+    border-radius: 9px;
+}
+.introjs-helperNumberLayer{
+    display: none;
+}
+.introjs-helperLayer{
+    border-radius: 9px;
+}
+.introjs-donebutton{
+    color: white !important;
```

Comment:
Let's talk about the !important's! There is usually a way to get around using them, especially by being more specific. There is some good information here if you want to read some more about why to avoid it and some other options: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity. I'm not sure how possible this is because you're editing css provided by the library, but it is worth an attempt! Let me know if you can't figure out how and I can see if it is possible.

Reply:
Unfortunately other than using IDs it looks like using !important is the best way to override it.
![image](https://user-images.githubusercontent.com/45320827/92989419-3b994080-f4a2-11ea-9e6c-349d38f7d19d.png)